src/pkg/cli/compose/loader.go

```diff
@@ -90,46 +109,25 @@ func getDefaultProjectOptions(composeFilePath string, extraOpts ...cli.ProjectOp
 	return projOpts, nil
 }
 
-func getComposeFilePath(userSpecifiedComposeFile string) (string, error) {
-	// The Compose file is compose.yaml (preferred) or compose.yml that is placed in the current directory or higher.
-	// Compose also supports docker-compose.yaml and docker-compose.yml for backwards compatibility.
-	// Users can override the file by specifying file name
-	const DEFAULT_COMPOSE_FILE_PATTERN = "*compose.y*ml"
-
-	path, err := os.Getwd()
+func findDefaultComposeFilePath() (string, error) {
+	opts := []cli.ProjectOptionsFn{
```

Comment:
Should we add the options from the default options before `cli.WithDefaultConfigPath`?
I am not sure if:
```
cli.WithWorkingDirectory(workingDir),
cli.WithEnvFiles(), 
cli.WithDotEnv,
cli.WithConfigFileEnv,
```
Has any side effect on the options to alter the result of `WithDefaultConfigPath`

Reply:
Your question prompted me to walk through their code again, and I believe the only thing we lose with my diff is the ability to configure the compose file with the `COMPOSE_FILE` env var.

That said, I had originally factored out this separate call to `cli.NewProjectOptions` so that we could inspect the the result of the `compose-go` code to find the default compose file before we tried to actually initialize the project options. Now that we aren't trying to return an error in the case of two default compose files being present, I think we could go back to only calling this once.

Reply:
There is a tricky subtlety to watch out for here though, and that is setting the `workingDir` so that the various environment files can be loaded.

If we pass in an explicit compose path, we set the working directory to be `filepath.Dir(composeFilePath)`, but if we use a default compose path from a parent directory, we also need to make sure we set the `workingDir` to that parent directory.

Reply:
I've decided to more closely mirror the way docker compose initializes its project options to minimize divergence by configuring the loader options directly from the cli flags here:

https://github.com/DefangLabs/defang/blob/18dcbfb44d3379d3e5d197329b6ebb72519a2467/src/cmd/cli/command/commands.go#L1263-L1269

To mirror the docker compose code here:

https://github.com/docker/compose/blob/d3d378b92af6eb02043c79db4a0757eb405a0692/cmd/compose/compose.go#L206-L217

Then uncommenting `cli.WithDefaultConfigPath` here:

https://github.com/DefangLabs/defang/blob/9fe354818fed38d7502a8d2ec56904772de5fdca/src/pkg/cli/compose/loader.go#L80

Brings us back in line with the docker compose code to normalize project options here:

https://github.com/docker/compose/blob/d3d378b92af6eb02043c79db4a0757eb405a0692/cmd/compose/compose.go#L367-L387

In doing so, I've added support for passing multiple compose files with `-f`, and also set up a nice foundation supporting the following additional flags, which I will complete in a followup to this PR.

```
      --env-file stringArray       Specify an alternate environment file
      --profile stringArray        Specify a profile to enable
      --project-directory string   Specify an alternate working directory
                                   (default: the path of the, first specified, Compose file)
  -p, --project-name string        Project name
```